starcatalogue/tasks.py

```diff
@@ -3,6 +3,7 @@
 import urllib
 import yaml
 import zipfile
+import json
```

Comment:
```suggestion
import ujson as json
```

ujson is faster (we'll also need to install it with pipenv).